.env.example

```diff
@@ -1,3 +1,8 @@
 DATABASE_URL=
 SENDGRID_API_KEY=
 HOST=
+FRONT_HOST=
+
+WALLET_ADDRESS=
+WALLET_MNEMONIC=
```

Comment:
It's better remove these 2, we don't need a mnemonic and the public address can be generated from the private key.